### PR TITLE
Make transient nature of consent more explicit in Perl6 version

### PR DESCRIPTION
**Pull request by iarna**
Plus make it a bit more idiomatic

### CODE REVIEW

File: perl6/lib/Cutie.pm [hunk 1]
```diff
@@ -1,3 +1,3 @@
 role Cutie {
-  has Bool $.accept-hugs is rw = False;
+  method accept-hugs { … }
```
Comment (zkat) [inline perl6/lib/Cutie.pm:2]: omg really? You need to use the ellipsis character in Perl6?! Or it at least accepts it?

Comment (iarna) [inline perl6/lib/Cutie.pm:2]: haha, yes! That's how you mark a place holder method that has to be defined by any class using the role. (You can also use `...`.) Perl6 has a bunch of unicode operators with optional ascii representations.

Comment (iarna) [inline perl6/lib/Cutie.pm:2]: Ellipsis characters are also valid constructor for lazy list objects, eg `my @laaazy = 1…Inf;`

Comment (zkat) [inline perl6/lib/Cutie.pm:2]: I like this.

My instinct is to just expect it to eventually converge on APL :P

Comment (iarna) [inline perl6/lib/Cutie.pm:2]: I just assume that's everyone's goals. APL, but with more emoji.

**Pull request merged**